### PR TITLE
rbind.sf doesn't preserve the crs of its input

### PR DESCRIPTION
**Pull request by faridcher**

### CODE REVIEW

File: R/sf.R [hunk 1]
```diff
@@ -249,7 +249,7 @@ print.sf = function(x, ..., n =
 #' @export
 rbind.sf = function(..., deparse.level = 1) {
 	ret = base::rbind.data.frame(...)
-	st_geometry(ret) = do.call(st_sfc, st_geometry(ret))
+	st_geometry(ret) = do.call(st_sfc, c(st_geometry(ret),crs=st_crs(st_geometry(ret))$epsg))
```
Comment (edzer) [inline R/sf.R]: using `crs = st_crs(reg)` makes it more compact and more general.

Comment (faridcher) [inline R/sf.R]: `crs = st_crs(reg)`  doesn't work. throws another error. `crs = st_crs(reg)$epsg` works fine.

Comment (edzer) [inline R/sf.R]: I meant `crs = st_crs(ret)`.

Comment (faridcher) [inline R/sf.R]: I see. that's a typo. `st_sfc` is supposed to received class type `crs` which is evident from `inherits(crs, "crs")`. But this line `u = unique(sapply(lst, class)[1,])` raises an error!

Comment (faridcher) [inline R/sf.R]: try is with `nc` dataset:
```
nc <- st_read(system.file("shape/nc.shp", package="sf"), "nc", crs = 4267)
ret = base::rbind.data.frame(nc)
st_geometry(ret) = do.call(st_sfc, c(st_geometry(ret),crs=st_crs(ret)))
```

Comment (edzer) [inline R/sf.R]: replace `c()` with `list()`.

**Pull request closed without merging**